### PR TITLE
Fix for hdlbits.01xz.net

### PR DESCRIPTION
**Pull request by linsui**
E.g. https://hdlbits.01xz.net/wiki/Xnorgate

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6533,6 +6533,13 @@ INVERT
 
 ================================
 
+hdlbits.01xz.net
+
+INVERT
+.image
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6539]:
```suggestion
CSS
.image {
    background-color: white !important;
    filter: brightness(50%) sepia(40%) !important;
}
```

**Comment by linsui** [comment]
@Gusted Anything else can I do?

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (linsui) [reply src/config/dynamic-theme-fixes.config]: This doesn't work well.

https://hdlbits.01xz.net/wiki/Fsm1

image

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Hmm weird, seems like this has a `.image` wrapper and thus the class isn't on the `img` element(which it has to be in order to work), I think you can change it to:

`img.image, .image > img`

Comment (linsui) [reply src/config/dynamic-theme-fixes.config]: It works well now, thanks!

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -6533,6 +6533,16 @@ INVERT
 
 ================================
 
+hdlbits.01xz.net
+
+CSS
+.image {
```
Comment (linsui) [inline src/config/dynamic-theme-fixes.config:6539]:
```suggestion
img.image, .image > img {
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**